README.md

```diff
@@ -0,0 +1,198 @@
+# GCP Dataflow Example Project
+
+## Introduction
+
+This is a simple stream processing example project, written in Scala and using Google Cloud Platform's services/APIs:
+
+- Dataflow for data processing
+- Pub/Sub for communication
+- Bigtable for storage
+
+**Running this requires a Google account and enabling GCP services, which will incur certain costs**
+
+We assume you have [pyinvoke](http://www.pyinvoke.org/) installed, as well as the [Google Cloud Python SDK](https://pypi.python.org/pypi/google-cloud), in order for the helper script to work.
```

Comment:
Is there any python version requirement? 
`pip install google-cloud` doesn't work on 3.5 and 3.6 as `google-cloud` has notable incompatibility with `setuptools=<33.0` I don't think its worth to mention, but if we're going to add FAQ/Troubleshooting setion - this is a good candidate to add.